systemtests/src/main/java/io/enmasse/systemtest/info/TestInfo.java

```diff
@@ -92,6 +90,17 @@ public boolean isTestShared() {
         return false;
     }
 
+    public boolean isTestIoT() {
+        for (String tag : getTags(tests.get(getCurrentTestIndex()))) {
+            if (TestTag.IOT_TAGS.stream().anyMatch(iotTag -> tag.equals(iotTag))) {
```

Comment:
Why not use a "set" here?

Reply:
I don't know if it's necessary, this method is usually called twice per test run...